### PR TITLE
update memory list

### PR DESCRIPTION
**Pull request by AnassM06**
Changes:
 you can search on country
sort on country

How to test: 
 go to search page and search on country
go to memory list and sort on country

Screenshots:
If applicable, add a screenshot or video here to show the new feature

### CODE REVIEW

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java [hunk 1]
```diff
@@ -75,8 +76,10 @@ private void filter(String text) {
         ArrayList<Memory> filteredlist = new ArrayList<>();
 
         for (Memory item : memories) {
+            Log.wtf("bla", "location:" + item.getCountryName(getContext()));
```
Comment (MrTheGood) [inline app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java:79]: This line is no longer necessary. Please remove it.

Comment (MrTheGood) [reply app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java:79]: We don't want to spam the logs unnecessarily.

**Review by MrTheGood — CHANGES_REQUESTED** [review]
Your code looks good! There is one small change though..

**Comment by Hicham010** [comment]
I asked you if you could also add years to your sorting. Can you still do this, Mr Anass?

**Review by Almightywesley — CHANGES_REQUESTED** [review]
come to school and then i can give you feedback

File: app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java (hunk 1 above)
Comment (Almightywesley) [reply app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java:79]: exactly

Comment (Hicham010) [reply app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java:79]: precise

**Review by Almightywesley — APPROVED** [review]

**Review by MrTheGood — APPROVED** [review]

**Pull request merged**